data-prepper-core/src/main/java/org/opensearch/dataprepper/core/parser/model/DataPrepperConfiguration.java

```diff
@@ -34,6 +34,7 @@
 public class DataPrepperConfiguration implements ExtensionsConfiguration, EventConfigurationContainer {
     static final Duration DEFAULT_SHUTDOWN_DURATION = Duration.ofSeconds(30L);
 
+    static final String DEFAULT_FAILURE_PIPELINE_NAME = "dlq";
```

Comment:
I think we should give this a different name.

```
default_failure_pipeline
```

Reply:
I think the name `dlq` already implies something since our DLQs are different from any other concept. So I think a failure pipeline is clearer. Maybe `default_dlq_pipeline` instead?